gocov/test.go

```diff
@@ -45,3 +146,66 @@ func runTests(args []string) error {
 	}
 	return convertProfiles(coverprofile.Name())
 }
+
+func runAllTests(args []string) (err error) {
+	pkgRoot, _ := os.Getwd()
+	pkg, err2 := relToGOPATH(pkgRoot)
+	if err2 != nil {
+		return err2
+	}
+	// TODO(maruel): This assumes this starts with "./". This is
+	// incorrect,someone could request to run test in a separate package.
+	requestedPath := filepath.Join(pkgRoot, args[0][:len(args[0])-3])
+	testDirs := goTestDirs(requestedPath)
```

Comment:
I don't think there's any point in converting the path to test dirs is there? Why not just run "go test" for each package that the argument expands to?